<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="NitinsDesktop007" failures="1" tests="4" name="Test" time="8.892" errors="0" timestamp="2025-08-03T18:50:14 IST">
  <testcase classname="api.test.userTest" name="testPostUser" time="3.477"/>
  <testcase classname="api.test.userTest" name="testGetUserByName" time="1.122"/>
  <testcase classname="api.test.userTest" name="testUpdateUserByName" time="2.12"/>
  <testcase classname="api.test.userTest" name="testDeleteUserByName" time="1.093">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.test.userTest.testDeleteUserByName(userTest.java:96)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testDeleteUserByName -->
</testsuite> <!-- Test -->
